tests/integrations/mcs/resourcemanager/resource_manager_test.go

```diff
@@ -78,6 +78,7 @@ func (suite *resourceManagerClientTestSuite) SetupSuite() {
 	suite.client, err = pd.NewClientWithContext(suite.ctx, suite.cluster.GetConfig().GetClientURLs(), pd.SecurityOption{})
 	re.NoError(err)
 	leader := suite.cluster.GetServer(suite.cluster.WaitLeader())
```

Comment:
do we need to check whether `suite.cluster.WaitLeader()` is empty?

Reply:
`leader` will be nil and be checked in L81